src/Backend/Core/Engine/Base/Config.php

```diff
@@ -127,6 +122,22 @@ public function isActionAvailable(string $action): bool
         return false;
     }
 
+    public function isActionAvailableForActionType(string $action, string $actionType): bool
+    {
+        // If this action is disabled for this type, continue on to the next type
```

Comment:
fixed it in e25c633
This was the result ob moving this code into its own method and I had forgotten to update the comments